### PR TITLE
Feat(cli) add version option (#18)

### PR DESCRIPTION
**Pull request by okam97**

### CODE REVIEW

File: packages/db/.prettierignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 docs/
 lib/
+README.md
```
[the remark this answers is not in the document]
Comment (okam97) [reply packages/db/.prettierignore]: I removed it now but testing errors at README.md even after running prettier. Should it be ignored?

[the remark this answers is not in the document]
Comment (birme) [reply packages/db/.prettierignore]: > I removed it now but testing errors at README.md even after running prettier. Should it be ignored?

No, the file it complains on should be corrected instead.

[the remark this answers is not in the document]
Comment (okam97) [reply packages/db/.prettierignore]: FIxed!

**Review by birme — APPROVED** [review]
LGTM now

**Pull request merged**